README.md

```diff
@@ -1,24 +1,61 @@
-# Digital Identity Credential Issuer
+# ipv-cri-toy-front
 
-# di-ipv-cri-toy-front
+[![Github Action: Unit Tests](https://github.com/govuk-one-login/ipv-cri-toy-front/actions/workflows/unit-tests.yml/badge.svg)](https://github.com/govuk-one-login/ipv-cri-toy-front/actions/workflows/unit-tests.yml?query=branch%3Amain)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=ipv-cri-toy-front&metric=coverage)](https://sonarcloud.io/summary/new_code?id=ipv-cri-toy-front)
```

Comment:
`.../badge.svg` returns `Not Found`

The URL for the unit tests returns `No results matched your search`
Unit tests are run as part of the Check PR workflow for PRs - it doesn't run on main. We can link to the coverage results from SonarCloud, it's neat, but I don't think we need to link to the unit tests workflow in the readme.

Reply:
Yeah, this PR predated a bunch of cleanup that happened with the GHAs

I've removed the unit test badge from this repo and the template repo too.